base/docs/basedocs.jl

```diff
@@ -1811,6 +1825,21 @@ Signed
     Unsigned <: Integer
 
 Abstract supertype for all unsigned integers.
+
+All unsigned integers are printed in hexadecimal, with prefix `0x`, 
+and can be entered in the same way.
```

Comment:
```suggestion
All unsigned integers are printed and can be parsed in hexadecimal, with prefix `0x`.
```

Reply:
Can you say why?

I guess I basically want to say you can type that in, but type with a keyboard is unclear when there are data types around. But it's a minor point since by default most printing can be pasted back in. That anything you type in is parsed seems like something people who know know, and who don't, don't need to.

Reply:
Ah! I missed the potential ambiguity of `type`/typing. "entered" reads awkwardly to me. Perhaps "written" instead?

Either 
```suggestion
All unsigned integers are printed in hexadecimal, with prefix `0x`, 
and can be written in the same way.
```
or
```suggestion
All unsigned integers are printed and can be written in hexadecimal, with prefix `0x`.
```